src/modules/memory_usage.rs

```diff
@@ -1,90 +1,72 @@
-use ansi_term::Color;
-
-use super::{Context, Module};
 use byte_unit::{Byte, ByteUnit};
 use sysinfo::{RefreshKind, SystemExt};
 
-/// Creates a module with system memory usage information
-pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
-    const DEFAULT_THRESHOLD: i64 = 75;
-    const DEFAULT_SHOW_PERCENTAGE: bool = false;
-    const RAM_CHAR: &str = "🐏 ";
+use super::{Context, Module, RootModuleConfig};
 
-    let mut module = context.new_module("memory_usage");
+use crate::configs::memory_usage::MemoryConfig;
 
-    if module.config_value_bool("disabled").unwrap_or(true) {
-        return None;
-    }
```

Comment:
Currently we do not have a high-order approach yet to decide if this module is disabled or not. This logic is required as the module is disabled by default.

Reply:
ok, I put it back in. I thought I had read somewhere that starship didn't load the module if it was `disabled`'d... maybe that was with the old module config